cmd/download.go

```diff
@@ -203,5 +203,5 @@ func downloadCurrentModel(ctx context.Context, absPath string, appId string, mod
 func init() {
 	RootCmd.AddCommand(downloadCmd)
 	downloadCmd.Flags().StringP("app", "a", "", "Which application's configuration or model to download. Can be given as the first positional argument.")
-	downloadCmd.Flags().String("model", "", "Download the model used by the application. Available machine learning frameworks are ort, tflite and coreml, if you specify all, all frameworks available for you will be downloaded. This feature is available on Enterprise plans (https://speechly.com/pricing).")
+	downloadCmd.Flags().String("model", "", "Specify the machine learning framework model to download. Available options are: ort, tflite, coreml and all. This feature is available on Enterprise plans (https://speechly.com/pricing)")
```

Comment:
@aukkonen @teelisyys the original help text felt clumsy after the change to how the model flag is used, so i rewrote it. Do you think this makes sense? Is "Specify the _machine learning framework model_ to download" the correct thing to say?

Reply:
I'd use "_...machine learning framework of the model..._" for extra clarity